geth/node/manager.go

```diff
@@ -98,12 +98,6 @@ func (m *NodeManager) startNode(config *params.NodeConfig) error {
 		log.Error("Failed to create an RPC client", "error", err)
 		return RPCClientError(err)
 	}
-	// populate static peers exits when node stopped
-	go func() {
```

Comment:
How are static peers from the config added after removing this fragment?

Reply:
See discussion in #708. It's in https://github.com/status-im/status-go/blob/develop/geth/node/node.go called in `MakeNode()` which is called in line 74 here.

Reply:
i think we had a misunderstanding :) that was link to my change https://github.com/dshulyak/status-go/blob/b407990303906357416a9aa6d506ebc092a919ff/geth/node/node.go#L120

it is different from what we have in develop branch https://github.com/status-im/status-go/blob/develop/geth/node/node.go#L119

Reply:
Aargh, `StaicNodes` vs `BootstrapNodes`. 😶

Reply:
also noticed that with that change I broke configuring bootstrap nodes with command line. i guess that bootstrap config should include both bootstrap nodes and static nodes

Reply:
Hm, I am confused now :D So we can get rid of this goroutine calling `PopulateStaticPeers` because we have `nc.P2P.BootstrapNodes = makeBootstrapNodes(config.BootClusterConfig.BootNodes)` now? Shouldn't it be `nc.P2P.StaticNodes`?

Reply:
Yes, `StaticNodes`. I'll tomorrow check the call paths, looking for missed settings.